engine/src/main/java/org/terasology/rendering/gltf/GLTFCommonFormat.java

```diff
@@ -65,7 +88,7 @@
             .registerTypeAdapter(TIntList.class, new TIntListDeserializer())
             .registerTypeAdapter(TFloatList.class, new TFloatListDeserializer())
             .registerTypeAdapter(Matrix4f.class, new Matrix4fDeserializer())
-            .registerTypeAdapter(Quat4f.class, new Quat4fDeserializer())
+            .registerTypeAdapter(Quaternionf.class, new Quat4fDeserializer())
```

Comment:
I'm not sure whether we should change the deserializers for the old types in place, or phase them out more gracefully. Are they used anywhere else, or just here for the glTF format?